src/clusterfuzz/stacktraces/constants.py

```diff
@@ -114,6 +114,8 @@
 FATAL_ERROR_REGEX = re.compile(r'#\s*Fatal error in (.*)')
 FATAL_ERROR_LINE_REGEX = re.compile(r'#\s*Fatal error in (.*), line [0-9]+')
 FATAL_ERROR_UNREACHABLE = re.compile(r'# un(reachable|implemented) code')
+FUZZER_EXIT_REGEX = re.compile(r'^\s*(?:#0|#1) 0x.*(?:fuzzer|fuzz/|/fuzz)',
+                               re.IGNORECASE)
```

Comment:
A more robust but more complicated solution would be to determine the file LLVMFuzzerTestOneInput is in (from the stacktrace) and use that to determine functions in the same file. But I'm fine with this approach.

Reply:
Thanks Jonathan! This PR is just the first out of a series targeting CF-filed issues arising from fuzzer implementation/infra errors, so I should have opportunity to further investigate the approach.